src/cmd/gz_TEST.cc

```diff
@@ -37,9 +37,6 @@
   #define pclose _pclose
 #endif
 
-// DETAIL_GZ_CONFIG_PATH is compiler definition set in CMake.
-#define GZ_CONFIG_PATH DETAIL_GZ_CONFIG_PATH
```

Comment:
I believe this was done intentionally so it's clear where the definition came from. That is: Define `DETAIL_GZ_CONFIG_PATH` in CMake and add the alias here with a comment.